<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AddEditCoffeeForm</class>
    <widget class="QDialog" name="AddEditCoffeeForm">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Добавить/Редактировать кофе</string>
        </property>
        <widget class="QWidget" name="centralwidget"/>

        <widget class="QLineEdit" name="nameEdit">
            <property name="geometry">
                <rect>
                    <x>130</x>
                    <y>20</y>
                    <width>200</width>
                    <height>24</height>
                </rect>
            </property>
        </widget>
        <widget class="QLineEdit" name="roastEdit">
            <property name="geometry">
                <rect>
                    <x>130</x>
                    <y>50</y>
                    <width>200</width>
                    <height>24</height>
                </rect>
            </property>
        </widget>
        <widget class="QCheckBox" name="groundCheck">
            <property name="geometry">
                <rect>
                    <x>130</x>
                    <y>80</y>
                    <width>100</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="text">
                <string>Молотый</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="descEdit">
            <property name="geometry">
                <rect>
                    <x>130</x>
                    <y>110</y>
                    <width>200</width>
                    <height>24</height>
                </rect>
            </property>
        </widget>
        <widget class="QDoubleSpinBox" name="priceSpin">
            <property name="geometry">
                <rect>
                    <x>130</x>
                    <y>140</y>
                    <width>100</width>
                    <height>24</height>
                </rect>
            </property>
            <property name="maximum">
                <double>10000.0</double>
            </property>
        </widget>
        <widget class="QSpinBox" name="volumeSpin">
            <property name="geometry">
                <rect>
                    <x>130</x>
                    <y>170</y>
                    <width>100</width>
                    <height>24</height>
                </rect>
            </property>
            <property name="maximum">
                <number>10000</number>
            </property>
        </widget>
        <widget class="QPushButton" name="saveBtn">
            <property name="geometry">
                <rect>
                    <x>150</x>
                    <y>220</y>
                    <width>100</width>
                    <height>30</height>
                </rect>
            </property>
            <property name="text">
                <string>Сохранить</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>